test/framework/src/main/java/org/elasticsearch/transport/AbstractSimpleTransportTestCase.java

```diff
@@ -1047,10 +1047,9 @@ public String executor() {
             .build());
 
         MockLogAppender appender = new MockLogAppender();
-        Loggers.addAppender(LogManager.getLogger("org.elasticsearch.transport.TransportService.tracer"), appender);
+        appender.start();
         try {
-            appender.start();
-
+            Loggers.addAppender(LogManager.getLogger("org.elasticsearch.transport.TransportService.tracer"), appender);
```

Comment:
Sorry, to be clear, I meant keeping the `appender.start();` in the `try` as it was, and instead of moving it up out of the `try` before the `Loggers.addAppender`, instead move the `Loggers.addAppender` below the `appender.start();`, both in the `try`.

Reply:
fixed in 50df262 :)